index.js

```diff
@@ -32,9 +29,7 @@ function createEngine(engineOptions) {
   // Since we're passing an object with jsx as the key, it'll override the rest.
   engineOptions = assign({}, DEFAULT_OPTIONS, engineOptions, {jsx: jsxOptions});
 
-  // Don't install the require until the engine is created. This lets us leave
-  // the option of using harmony features up to the consumer.
-  nodeJSX.install(engineOptions.jsx);
+  require('babel/register');
```

Comment:
I was originally passing `engineOptions.jsx.extension` through to babel, but I removed it while debugging something and forgot to put it back in.

That said, I hadn't considered using the `only` option, and only transforming files in the view directory.  I checked and it is possible to infer the view directory, the only difference is you have to defer registering babel until the first request is made, since that's the first time you can grab the view directory.

I think it makes sense to limit it to the view directory.  This means less `require.cache` clearing, and it just seems logical that only those files would be affected by a view engine.

I don't know where that leaves the extension option.  I'm not sure if keeping it around is overkill, since we'd already only be transforming the view folder.  My initial thought is it seems unlikely that you'd have multiple file types in your view directory and only want to transpile some of them, but maybe I'm wrong about that.